src/cortex/util.cljc

```diff
@@ -58,14 +58,20 @@
 
 
 (defn max-index
+  "Good for finding softmax winners and converting back from one-hot."
```

Comment:
`s/Good for//` superfluous... every function should be "good for" something. Should read as `Given a collection...`